### ISSUE TITLE
[Latest release] I can not fold/unfold attachments

### ISSUE THREAD

**Issue by chibenwa** [OpenPaaS-Suite/esn-frontend-inbox#448]
https://user-images.githubusercontent.com/6928740/117392520-70b83e80-af1c-11eb-9acf-5dd8bbfbfd9a.mp4

I click the fold/unflod side panel button without effect.

Mozilla Firefox 88.0

**Comment by Arsnael** [comment]
Hm I guess it's more about the definition of what we want here. I just checked with a mail where I have 4 or 5 attachments (so multi lines display as too many attachments to appear on one line.

When I play with the fold/unfold button, it switches between a reduced view of it with a scroll panel, or a full view of all attachments. I don't think I would see it as a bug, but more like maybe need to rethink this function

**Comment by chibenwa** [comment]
Then we should not show the button when attachments fits in one line

**Comment by fabienmoyon** [comment]
yes exactly but it's not so easy to know if attachments fits in one line or not because it depend of the size of your desktop. In one line you can have 1,2,3 or 4 attachments with the responsive design).
I will try later to fix by hide the collapse/expend icon if not needed

**Comment by fabienmoyon** [comment]
related to https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/347#issuecomment-793302969
